src/mixins/filing-template-mixin.ts

```diff
@@ -574,6 +574,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
    * @param entitySnapshot the latest entity snapshot
    */
   parseCorrectionFiling (filing: CorrectionFilingIF, entitySnapshot: EntitySnapshotIF = null): void {
+    // make a copy so snapshot objects are different from objects below
+    entitySnapshot = cloneDeep(entitySnapshot)
```

Comment:
I found some cases where the same object reference was used for a couple of things, which incorrectly updated this entity snapshot in the store. The snapshot should never change after this line of code!